pkg/process/procutil/process_linux.go

```diff
@@ -72,6 +72,15 @@ func WithReturnZeroPermStats(enabled bool) Option {
 	}
 }
 
+// WithCustomProcFSRoot confiugres the procfs directory that the probe reads from
+func WithCustomProcFSRoot(path string) Option {
```

Comment:
```suggestion
func WithProcFSRoot(path string) Option {
```

Suggestion, optional: it's a bit redundant to say it's custom

Reply:
I will say that it's overridden, because we do originally grab it from the `HOST_PROC` environment variable.